src/App.js

```diff
@@ -1,14 +1,25 @@
 import React from 'react';
-import MainDashboard from './components/Dashboard/MainDashboard/MainDashboard';
+import { createBrowserRouter, RouterProvider } from 'react-router-dom';
+
 import Header from './components/Header/Header';
+import Login from './components/Login/Login';
+import MainDashboard from './components/Dashboard/MainDashboard/MainDashboard';
+import Auth from './Auth/Auth';
+import UnauthorizedPage from './components/Unauthorized/UnauthorizedPage';
+
+const router = createBrowserRouter([
+  { path: '/', element: <Auth><MainDashboard /></Auth>},
+  { path: '/login', element: <Login />},
+  { path: '/unauthorized', element: <UnauthorizedPage />}
+]);
```

Comment:
please use in const in path routes as 
```suggestion
  { path: 'ROUTES.HOMEPAGE', element: <Auth><MainDashboard /></Auth>},
  { path: 'ROUTES.LOGIN', element: <Login />},
  { path: 'ROUTES.UNAUTORIZED', element: <UnauthorizedPage />}
]);
```